assets/app/views/settings.html

```diff
@@ -0,0 +1,83 @@
+<div ng-controller="ProjectController">
+  <project-page>
+    <div ng-controller="SettingsController">
+      <h1>Project Settings</h1>
+      <alerts alerts="alerts"></alerts>
+      <h2>General information</h2>
+      <!-- TODO make these fields editable - probably with a cog icon that flips into inline-editing -->
+      <dl class="dl-horizontal left">
+        <dt>Display name:</dt>
+        <dd ng-if="project.displayName">{{project.displayName}}</dd>
+        <dd ng-if="!project.displayName"><em>No display name</em></dd>
+        <dt>Description:</dt>
+        <dd ng-if="project.metadata.annotations && project.metadata.annotations.description">{{project.metadata.annotations.description}}</dd>
```

Comment:
is `metadata.annotations.description` a standard annotation? If so, should we make a filter to do this for us?